docs/source/project_documentation/user_guide/project_structure.md

```diff
@@ -9,6 +9,39 @@ src/service_one/service_one_misc_definitions.yaml
 src/service_two/service_two.yaml
 ...
 ```
+src
+├── system
+│   └── system.yaml
+├── external
+|   ├── external_actors.yaml
+│   └── external_interface_messages.yaml
+├── service_one
+|   ├── service_one.yaml
+│   └── service_one_misc.yaml
+└── service_two
+    └── service_two.yaml
+
+Because AaC does not require your models to be located within the same directory or even within the same directory as your AaC package install, you can house your model files however is necessary and logical for your project needs. An example of this might look like:
```

Comment:
I see what you're saying here but I don't think it makes sense in general to put the model files within the same directory as the AaC install - i.e. a system path or a venv path.

```suggestion
Because AaC does not require your models to be located within the same directory, you can house your model files however is necessary and logical for your project needs. An example of this might look like:
```